client/dashboard/components/icon-list/index.stories.tsx

```diff
@@ -0,0 +1,97 @@
+import { Meta, StoryObj } from '@storybook/react';
+import { Icon } from '@wordpress/components';
+import { cog, page, layout } from '@wordpress/icons';
+import IconList from './index';
+
+const meta: Meta< typeof IconList > = {
+	title: 'client/dashboard/IconList',
+	component: IconList,
+	tags: [ 'autodocs' ],
+};
+
+export default meta;
+
+type Story = StoryObj< typeof IconList >;
+
+export const Default: Story = {
+	args: {
+		children: (
+			<>
+				<IconList.IconListItem
```

Comment:
Could this be simplified to avoid the redundant qualifier?


```suggestion
				<IconList.Item
```